app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -7,6 +7,7 @@ module HostManagedExtensions
 
       module Overrides
         def update(attrs)
+          return super unless self.content_facet.present?
```

Comment:
fun fact: Originally I just had `return unless`.. here. But since Foreman relies on the return value of `update`,

```rb
if @host.update(attributes)
```

it was processing it as an error even though there were no errors. I pulled out a few hairs before I figured that one out. :smile: